docker/policies/copy_from_without_from_alias_defined_previously.rego

```diff
@@ -0,0 +1,53 @@
+package appshield.dockerfile.DS018
+
+import data.lib.docker
+
+__rego_metadata__ := {
+	"id": "DS018",
+	"title": "COPY '--from' Without FROM Alias Defined Previously",
+	"version": "v1.0.0",
+	"severity": "HIGH",
+	"type": "Dockerfile Security Check",
+	"description": "COPY command with the flag '--from' should mention a previously defined FROM alias",
+	"recommended_actions": "Fix alias",
+	"url": "https://docs.docker.com/develop/develop-images/multistage-build/",
+}
+
+__rego_input__ := {
+	"combine": false,
+	"selector": [{"type": "dockerfile"}],
+}
+
+get_copy_arg[arg] {
+	copy := docker.copy[_]
+
+	arg := copy.Flags[x]
+	contains(arg, "--from=")
+	arg != "--from=0"
```

Comment:
What about `--from=1`, `--from=2`, and etc?

Reply:
fixed